commcare_connect/opportunity/tables.py

```diff
@@ -493,16 +493,21 @@ class Meta:
 
 
 class PaymentInvoiceTable(OpportunityContextTable):
+    amount = tables.Column(verbose_name="Amount")
     payment_status = columns.Column(verbose_name="Payment Status", accessor="payment", empty_values=())
     payment_date = columns.Column(verbose_name="Payment Date", accessor="payment", empty_values=(None))
     actions = tables.Column(empty_values=(), orderable=False, verbose_name="Pay")
+    exchange_rate = tables.Column(orderable=False, empty_values=(None,), accessor="exchange_rate__rate")
```

Comment:
nit: This empty_values param is needed only for the columns that are rendered using `render_{column_name}` methods.